pysensu_yelp/__init__.py

```diff
@@ -288,6 +290,13 @@ def send_event(
     :param sensu_host: The IP or Name to connect to for sending the event.
                        Defaults to the yocalhost IP.
 
+    :type component: list
+    :param component: Component(s) affected by the event.
+
+    :type description: str
+    :param description: Human readable text giving more context on the event.
+                        This is used to enrich the information sent to PagerDuty.
```

Comment:
Part of the design of this infrastructure was to remain as infra-agnostic as possible. This is why we use `page: true` instead of `send_to_pagerduty:`, etc.

Can you leave out the fact that it is used in pagerduty? If this is provided I would like to use it in all our handlers, not just PD

Reply:
Sounds good! I will write up a generic statement here.